web/packages/teleterm/src/ui/Search/pickers/PickerContainer.ts

```diff
@@ -24,10 +24,11 @@ export const PickerContainer = styled.div`
   z-index: 1000;
   font-size: 12px;
   color: ${props => props.theme.colors.text.contrast};
-  background: ${props => props.theme.colors.levels.surface};
-  box-shadow: 8px 8px 18px rgb(0, 0, 0, 0.56);
+  background: ${props => props.theme.colors.levels.elevated};
+  box-shadow: ${props => props.theme.boxShadow[1]};
   border-radius: ${props => props.theme.radii[2]}px;
-  border: 1px solid ${props => props.theme.colors.action.hover};
+  // border in the same color as the background to prevent layout shifting
```

Comment:
Could you go into more detail here? I was able to understand what this does by removing the style but I feel like the comment alone doesn't tell the whole story. As far as I understand, this has to do with PickerContainer wrapping the input when the search bar is open, so we have to use a border of the same width to keep the search bar layout from shifting when switching between open and closed state, right?

Reply:
That's the exact reason, I will include it in the comment.